src/App.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable*/
+/* eslint-disable */
 import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import { Provider } from 'react-redux';
```

Comment:
### Error
![errors](https://user-images.githubusercontent.com/93039359/187048275-20a66bbb-db22-44b7-8b97-5c3fc473d785.png)

- [ ] Your application is broken and not showing anything other than these errors please fix them to make your application work :+1: